cranelift/interpreter/src/interpreter.rs

```diff
@@ -119,13 +119,16 @@ impl Interpreter {
     /// Interpret a single [Block] in a [Function].
```

Comment:
pre-existing: this comment is lying just a bit, right? It interprets from a given block up to the function's return, if I'm not mistaken.

Reply:
Good call... changed.